src/applications/disability-benefits/2346/helpers/index.js

```diff
@@ -73,11 +75,11 @@ export function validateZIP(errors, zip) {
 }
 
 export const showNewAddressForm = formData =>
-  Object.values(formData?.newAddress).every(entry => Boolean(entry));
+  formData?.selectedAddress === newAddressField;
 
 export const getRadioLabelText = (formData, name) => {
   const address = formData?.[name] || {};
   return Object.values(address)
     .filter(entry => Boolean(entry))
-    .join(' ');
+    .join(', ');
```

Comment:
This wont format the address properly
![Screen Shot 2020-04-13 at 5 54 37 PM](https://user-images.githubusercontent.com/6165421/79164842-e56b8880-7daf-11ea-909e-deb658e51a49.png)